example-configs/integreat-cms.ini

```diff
@@ -115,6 +115,10 @@ EMAIL_USE_SSL = False
 [linkcheck]
 # Whether link check should be disabled [optional, defaults to False]
 LINKCHECK_DISABLE_LISTENERS = False
+# URL types that are not supposed to be shown in the link list (e.g. phone numbers and emails)
+LINKCHECK_IGNORED_URL_TYPES =
```

Comment:
Do I understand correctly that these settings should be applied automatically, or do I have to do something extra for that?
Currently the value of settings.LINKCHECK_IGNORED_URL_TYPES for me is resolved as [].

Reply:
The .ini file in the repository is just an example config to illustrate what values are possible.
I added this so we can quickly return to ignoring emails and phone numbers if it's too confusing for the users, but per default I wouldn't ignore them, that's why I set the empty list as default value in the settings file.

Reply:
Ah, okay, thanks. I thought this config is loaded somehow